src/views/clusteroverview/SettingsTab/ClusterTab/components/TemplatesProjectSection/TemplatesProjectSection.tsx

```diff
@@ -51,7 +57,7 @@ const TemplatesProjectSection: FC = () => {
     },
   );
 
-  const [hyperConverge, hyperLoaded, hyperLoadingError] = useHyperConvergeConfiguration();
+  const [hyperConverge, hyperLoaded] = hyperConvergeConfiguration;
```

Comment:
What is the reason of removing `hyperLoadingError` from here? Same for other changes in this PR..

Reply:
i moved the hook to central location instead of many call that each component did, so error of hco I also display in a single place in upper component, I've mention it in the description